tools/unique_regions/unique_regions.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+
+def remove_first_line(s):
+    return s[s.find('\n')+1:s.rfind('\n')]
+
+def overlap(infile):
+    overlapping = ""
+    unique = ""
+    prev_chromo = ""
+    prev_start = 0
+    prev_end = 0
+
+    #iterating the bed file, line by line
+    for line in open(infile, 'r'):
+        split_line = line[:-1].split("\t")
+        chromo = str(split_line[0])
+        start = int(split_line[1])
+        end = int(split_line[2])
+
+        #verifying whether the two successive regions are on the same chromosome,
+        #if not, the comparison will be skipped
+        if chromo != prev_chromo:
+            prev_chromo = chromo
```

Comment:
These blocks are the same/similar than the blocks below. Maybe you can put this in a small helper function to reduce code and DRY.